src/components/Message.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import propTypes from 'prop-types';
+
+function Message({ message, guest_id }) {
+  if (!message.sender) debugger;
+  return (
```

Comment:
we should probably take `debugger` out before pushing to production

Reply:
got rid of it